owntone@.service.in

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=DAAP/DACP (iTunes), RSP, MPD server, with AirPlay and Remote - %I
+Documentation=man:owntone(8)
+Requires=network.target local-fs.target avahi-daemon.socket
+After=network.target sound.target remote-fs.target pulseaudio.service
+
+[Service]
+ExecStart=@sbindir@/owntone -f -c /etc/owntone-%I.conf
+SyslogIdentifier=owntone-%I
+
+# Constrain the upper limit of memory/swap that can be used; this prevents
```

Comment:
The trouble with repeating the comments from owntone.service is that in the future some maintainer, maybe even myself, will update owntone.service and forget to update this. So I would ask you to remove the comments and just have one comment at the top, which should be an explanation of what the purpose of this file is, and then a reference to the normal service file. Might also be good with a note in the normal service file that reminds future maintainers that updates must also be carried over to this file.

Reply:
So noted.  I added a comment at the top of each service file referencing the other.